.storybook/main.ts

```diff
@@ -16,6 +23,16 @@ const config: StorybookConfig = {
         path.join("@storybook/addon-interactions", "package.json")
       )
     ),
+    {
+      name: '@storybook/addon-docs',
+      options: {
+        mdxPluginOptions: {
+          mdxCompileOptions: {
+            remarkPlugins: [remarkGfm],
+          },
+        },
+      },
+    },
```

Comment:
https://storybook.js.org/docs/react/writing-docs/mdx#lack-of-github-flavored-markdown-gfm